### ISSUE TITLE
figure out how to test dynamically generated form field on signup

### ISSUE THREAD

**Issue by oorion** [oorion/tutor_connect#12]

**Comment by thatrubylove** [comment]
care to give more info on this? I might be able to help.

interesting project

**Comment by oorion** [comment]
Sure.  We thought it would be useful if there was an easier way to connect tutors with people who needed tutoring in their area.  The app was originally being designed to connect high school students who were willing to tutor with an elementary school teacher who's students needed tutoring and help them to organize an after-school meeting to give younger students more one on one attention.  The idea could be extended as well.

**Comment by thatrubylove** [comment]
I mean about this issue.... :D details

Jim OKelly
@thatrubylove

Advanced Ruby musings @ RubyLove
https://www.rubylove.io

Expert mentor @ CodeMentor
https://www.codementor.io/thatrubylove

What does Beautiful, Functional, Testable Ruby code look like??? Glad you
asked:
http://rubylove.io/ruby_programming_manifesto/

**Comment by oorion** [comment]
We got the page working as intended but never figured out how to test it.  For instance, when a user selects teacher from the signup dropdown, it asks them which school they teach at.  If they select student, it asks them what school they attend.

**Comment by oorion** [comment]
I suppose using capybara to click the select box and test what the page says would suffice.

**Comment by thatrubylove** [comment]
yeah, that would be capy or cuke work and you would need a web driver like
webkit to power the javascript

Jim OKelly
@thatrubylove

Advanced Ruby musings @ RubyLove
https://www.rubylove.io

Expert mentor @ CodeMentor
https://www.codementor.io/thatrubylove

What does Beautiful, Functional, Testable Ruby code look like??? Glad you
asked:
http://rubylove.io/ruby_programming_manifesto/
